infra/helper.py

```diff
@@ -135,15 +135,17 @@ def corpus(self):
 def main():  # pylint: disable=too-many-branches,too-many-return-statements
   """Gets subcommand from program arguments and does it. Returns 0 on success 1
   on error."""
-  os.chdir(OSS_FUZZ_DIR)
```

Comment:
Why don't we need this anymore?

Reply:
This is still here, just moved to line 145.

Reply:
ACK